Tests/Controller/AuthorizeControllerTest.php

```diff
@@ -221,7 +221,7 @@ public function testAuthorizeActionWillThrowAccessDeniedException()
 
         $this->setExpectedException(
             AccessDeniedException::class,
-            'This user does not have access to this section.');
+            'This user does not have access to this section.')
```

Comment:
this change looks unrelated

Reply:
Sure but I don't think it needs a separate commit as it has no impact.